napari/layers/points/points.py

```diff
@@ -16,10 +16,12 @@
 )
 
 import numpy as np
+import numpy.typing as npt
 import pandas as pd
 from psygnal.containers import Selection
 from scipy.stats import gmean
 
+from napari.components.dims import Dims
```

Comment:
I think this is causing the circular import error?
Should it go under `if TYPE_CHECKING:`?